### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by doevent** [terry3041/pyChatGPT#45]
Error: list index out of range

Version: `0.3.9.4`

```

File "pyChatGPT.py", line 395, in send_message
    response = self.driver.find_elements(
IndexError: list index out of range
```

**Comment by 872226263** [comment]
same problem, i post my soultion in https://github.com/terry3041/pyChatGPT/issues/46

**Comment by scaryama** [comment]
My solve is modify 'pyChatGPT.py' file
"//div[starts-with(@Class, 'markdown prose break-words')]"
↓
"//div[starts-with(@class, 'markdown prose w-full break-words')]"

**Comment by terry3041** [comment]
Should be fixed in v0.3.9.5.
